__tests__/main.test.ts

```diff
@@ -332,7 +328,8 @@ describe('queries', () => {
                     node: {
                       id: 'MDExOlB1bGxSZXF1ZXN0NTc4ODgyNDUw',
                       number: 7,
-                      mergeable: 'MERGEABLE',
+                      mergeable: MergeableState.MERGEABLE,
+                      mergeStateStatus: MergeStateStatus.CLEAN,
```

Comment:
These should be string values https://docs.github.com/en/graphql/reference/enums#mergestatestatus